src/domain/notification/create/feature.ts

```diff
@@ -0,0 +1,23 @@
+
+import logger from '^/integrations/logging/module';
+
+import type { Requester } from '^/domain/authentication/types';
+
+import type { Types } from '../definitions';
+import createData from './createData';
+import insertData from './insertData';
+
+export default async function feature(requester: Requester, type: Types, receiverId: string, postId?: string, reactionId?: string): Promise<void>
+{
+    const data = createData(type, requester.id, receiverId, postId, reactionId);
```

Comment:
Currently, notifications are created for everybody, including your own. I think we should ignore notifications where requester is the receiver. Do you agree?